server/mainPr/src/main/java/com/team23/mainPr/Domain/Comment/Service/CommentService.java

```diff
@@ -0,0 +1,66 @@
+package com.team23.mainPr.Domain.Comment.Service;
+
+import com.team23.mainPr.Domain.Comment.Dto.Request.CreateCommentEntityDto;
+import com.team23.mainPr.Domain.Comment.Dto.Request.UpdateCommentEntityDto;
+import com.team23.mainPr.Domain.Comment.Dto.Response.CommentEntityResponseDto;
+import com.team23.mainPr.Domain.Comment.Entity.Comment;
+import com.team23.mainPr.Domain.Comment.Mapper.CommentMapper;
+import com.team23.mainPr.Domain.Comment.Repository.CommentRepository;
+import com.team23.mainPr.Global.DefaultTimeZone;
+import com.team23.mainPr.Global.Dto.ChildCommonDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import static com.team23.mainPr.Global.Enum.ChildCommonDtoMsgList.FAIL;
+import static com.team23.mainPr.Global.Enum.ChildCommonDtoMsgList.SUCCESS;
+
+@Service
+@RequiredArgsConstructor
+public class CommentService {
+
+    private final CommentMapper commentMapper;
+    private final DefaultTimeZone defaultTimeZone;
+    private final CommentRepository commentRepository;
+
+    public ChildCommonDto<CommentEntityResponseDto> createCommentEntity(CreateCommentEntityDto createCommentEntityDto) {
```

Comment:
```suggestion
    public ChildCommonDto<CommentEntityResponseDto> createCommentEntity(CreateCommentEntityDto dto) {
```
`nit` 장황한 코드를 축약할 수 있습니다.

Reply:
사용되는 dto 타입을 알기 좋겠다고 생각했는데 바꾸도록 하겠습니다 ㅠ